zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -120,6 +140,34 @@ final case class Cookie(
     cookie.flatten.mkString("; ")
   }
 
+  /**
+   * Signs cookie content with a secret
+   */
+  private def sign(secret: String, content: String): String = {
+    try {
+      val sha256    = Mac.getInstance("HmacSHA256")
+      val secretKey = new SecretKeySpec(secret.getBytes(), "RSA")
+      sha256.init(secretKey)
+      val signed    = sha256.doFinal(content.getBytes())
+      val mda       = MessageDigest.getInstance("SHA-512")
+      content + '.' + getEncoder.encodeToString(mda.digest(signed))
+    } catch {
+      case _: Exception => content
```

Comment:
Returning the same content is quite dangerous and difficult to debug. I would rather have a failure on the server this way.

Reply:
But if we are not able to sign the cookie then it should return cookie(without sign)?